storage/rocksdb/rdb_datadic.cc

```diff
@@ -4095,6 +4110,11 @@ void Rdb_ddl_manager::remove(Rdb_tbl_def *const tbl,
   const rocksdb::Slice tkey((char *)buf, pos);
   m_dict->delete_key(batch, tkey);
 
+  uchar key_buf[Rdb_key_def::INDEX_NUMBER_SIZE * 3] = {0};
+  m_dict->get_key_from_index_id((uchar *)key_buf, gl_index_id);
+  auto auto_increment_key = rocksdb::Slice((char *)key_buf, sizeof(key_buf));
+  m_dict->delete_key(batch, auto_increment_key);
```

Comment:
I thought we discussed Rdb_dict_manager::delete_index_info() would be a better place to remove the auto_increment key. There would be no need to add the gl_index_id argument to this function.